### PR TITLE
Please review the dg-dev pull request

### PR DESCRIPTION
**Pull request by darclay**
This includes the creation of three components: 
1. PostCard
2. ExploreCard
3. ExploreContainer

### CODE REVIEW

File: client/src/components/CenterPanel/components/ExploreContainer/ExploreContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    {
+      tagImg: "https://i.imgur.com/zS6a6UN.png",
+      tagName: "AID HOMELESS"
+    }
+  ]
+  return (
+    <div className="ExploreContainerDiv">
+      <h1 className="exploreTitle">EXPLORE TAGS</h1>
+      <div className="exploreFlexContainer">
+        <ExploreCard tags ={tags}/>
```
Comment (gooliajulia) [inline client/src/components/CenterPanel/components/ExploreContainer/ExploreContainer.jsx:39]: My understanding of the Container + Card dynamic is that the Card represents a single Card, and the Container does the mapping of the cards. @johnnydomingo + @thepaiger curious what you think!

Comment (gooliajulia) [inline client/src/components/CenterPanel/components/ExploreContainer/ExploreContainer.jsx:39]: My understanding of the Container + Card dynamic is that the Card represents a single Card, and the Container does the mapping of the cards. @johnnydomingo + @thepaiger curious what you think!

File: client/src/components/CenterPanel/components/PostCard/PostCard.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+              <li><img className="memberIcons first" src="https://i.imgur.com/7hH5dsK.png" alt="joined member" /></li>
+              <li><img className="memberIcons second"  src="https://i.imgur.com/CfuyAtJ.png" alt="joined member" /></li>
+              <li><img className="memberIcons third"  src="https://i.imgur.com/vnZUCFM.png" alt="joined member" /></li>
+            </ul>
+          </div>
+        </div>
+      
+      </div>
+    </div>
+  )
```
Comment (gooliajulia) [inline client/src/components/CenterPanel/components/PostCard/PostCard.jsx:52]: I think you did a great job with the Post Card! I love the separation by comments to make it easier come back to and work on when necessary. Awesome job with the CSS too, I feel like this is one of our trickiest components styling wise and you crushed it

Comment (gooliajulia) [inline client/src/components/CenterPanel/components/PostCard/PostCard.jsx:52]: I think you did a great job with the Post Card! I love the separation by comments to make it easier come back to and work on when necessary. Awesome job with the CSS too, I feel like this is one of our trickiest components styling wise and you crushed it

**Review by gooliajulia — COMMENTED** [review]
You're a machine. Curious to see what Paige + Johnny say about the container / card comment but otherwise everything looks great!

One note - It is generally best practice to keep node modules out of the repo and pull requests since they're local files and can be pretty dense (adding hundreds of files here to review and making it harder to focus on the handful of files we do need to check out).

I BELIEVE the way to avoid this is adding node modules to the .gitignore file like this repo outlines:

https://github.com/StephenGrider/FullstackReactCode/issues/3

We can check this out together at tonights meeting

File: client/src/components/CenterPanel/components/ExploreContainer/ExploreContainer.jsx (hunk 1 above)
Comment (johnnydomingo) [reply client/src/components/CenterPanel/components/ExploreContainer/ExploreContainer.jsx]: @gooliajulia + @thepaiger I see this the same as you - the container surrounds EACH card!

Comment (thepaiger) [reply client/src/components/CenterPanel/components/ExploreContainer/ExploreContainer.jsx]: Agreed!

Comment (thepaiger) [reply client/src/components/CenterPanel/components/ExploreContainer/ExploreContainer.jsx]: Agreed, just like postCard

Comment (darclay) [reply client/src/components/CenterPanel/components/ExploreContainer/ExploreContainer.jsx]: I think I have this right now.  Can someone please review? Thanks

**Review by johnnydomingo — APPROVED** [review]
Looking good Dave!

**Pull request merged**